production/frontend/src/tests/components/injectables/calender.test.js

```diff
@@ -52,7 +52,11 @@ describe("Calender render Tests", () => {
       </WebsocketContextProvider>
     </StateContextProvider>);
 
-    expect(screen.getByText("26. June")).toBeVisible();
+    let expectedDate = "26. June"; //Test taking account for local settings used by date
+    if(Intl.DateTimeFormat().resolvedOptions().locale = "da-DK") {
```

Comment:
This looks like an assignment? Is this correct?

Also Turnaries / lambda's are your friend Try:
const expectedDate = Int1.DateTimeFormat().resolvedOptions().locale === "da-DK" ? "26. juni" : "26. June";

Reply:
No it's not supposed to be an assignment, thanks for catching that.